configuration/messageproperties/messages_en.properties

```diff
@@ -1529,13 +1529,20 @@ pihcore.mealsPerDay=Meals per day
 pihcore.numberPerDay=Number per day
 zl.socioEconomic.monthsLivingInHaiti=Number of months residing in Haiti
 
-
-pihcore.mentalhealth.generalAppearance =General appearance wnl
-pihcore.mentalhealth.speech=Speech wnl
-pihcore.mentalhealth.cognitive=Cognitive function wnl
+pihcore.mentalhealth.status=Status
+pihcore.mentalhealth.generalAppearance =General appearance within normal limits
+pihcore.mentalhealth.speech=Speech within normal limits
+pihcore.mentalhealth.cognitive=Cognitive function within normal limits
 pihcore.mentalhealth.mood=Mood disorder
 pihcore.mentalhealth.muscleTone=Muscle tone and strength
 
+pihcore.mentalhealth.PoorIntrospection =Poor introspection
```

Comment:
Remove space.

I recommend changing the text to "Introspection" with yes/no or normal/abnormal.  That matches the concept.